hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/conf/DatanodeRatisServerConfig.java

```diff
@@ -54,14 +54,16 @@ public void setRequestTimeOut(Duration duration) {
   }
 
   @Config(key = "watch.timeout",
-      defaultValue = "180s",
+      defaultValue = "10s",
       type = ConfigType.TIME,
       tags = {OZONE, DATANODE, RATIS},
       description = "The timeout duration for watch request on Ratis Server. " +
           "Timeout for the watch request in Ratis server to acknowledge a " +
-          "particular request is replayed to all servers."
+          "particular request is replayed to all servers. It is highly recommended " +
+          "for the timeout duration to be strictly shorter than Ratis client watch timeout " +
+          "(hdds.ratis.raft.client.rpc.watch.request.timeout)."
   )
-  private long watchTimeOut = Duration.ofSeconds(180).toMillis();
+  private long watchTimeOut = Duration.ofSeconds(10).toMillis();
```

Comment:
BTW, it looks like this getWatchTimeOut() is not called anywhere.

Reply:
Thanks for the review. Yes, `getWatchTimeOut` is not called anywhere, which was confusing me at first.

However, the configuration `hdds.ratis.raft.server.watch.timeout` will be processed in `RatisHelper#createRaftServerProperties`, which will trim the `hdds.ratis.` prefix and set it to the Ratis properties. Checked previously that changing the configuration actually updated the Ratis server watch timeout configuration.

This means that some calls to `DatanoteRatisServerConfig` in `XceiverServerRatis` might not be necessary (e.g. , `DatanoteRatisServerConfig#getClientPoolSize`). However, we need to double check before removing these calls since some of the `DatanodeRatisServerConfig` property keys do not correspond to any Raft server property keys and need to be set explciitly (e.g. `DatanodeRatisServerConfig#getStreamRequestThreads`).

We can remove the `getWatchTimeOut` if needed.

Reply:
I see. Thanks for the explanation. It's OK to keep the getWatchTimeOut.